components/prism-bicep.js

```diff
@@ -0,0 +1,29 @@
+// based loosely upon: https://github.com/Azure/bicep/blob/main/src/textmate/bicep.tmlanguage
+Prism.languages.bicep = {
+	'comment': [
+		{
+			// multiline comments eg /* ASDF */
+			pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+			lookbehind: true,
+			greedy: true
+		},
+		{
+			// singleline comments eg // ASDF
+			pattern: /(^|[^\\:])\/\/.*/,
+			lookbehind: true,
+			greedy: true
+		}
+	],
+	'string': {
+		// this doesn't handle string interpolalation or multiline strings as yet
+		pattern: /(')(?:\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
```

Comment:
1. Backreferences for a single character (`'`) are unnecessary and expensive (about 3x slower).
2. The `\\(?:\r\n|[\s\S])` is incorrect. [Strings [...] must be declared on a single line](https://github.com/Azure/bicep/blob/main/docs/spec/bicep.md#strings).

```suggestion
		pattern: /'(?:\\.|[^'\\\r\n])*'/,
```